apps/staking/hooks/useStakes.tsx

```diff
@@ -81,9 +95,10 @@ export function useStakes(overrideAddress?: Address, autoUpdateIntervalOverride?
     if (contractsErr) logger.error(contractsErr);
 
     //Minimum time in seconds that a node can go from "joining" to "exited"
-    const nodeMinLifespan = SESSION_NODE_TIME(chainId).EXIT_REQUEST_TIME_SECONDS;
+    const nodeMinLifespan = SESSION_NODE.INITIAL_DOWNTIME_CREDITS_HOURS * 60 * 60 * 1000;
```

Comment:
Why make `INITIAL_DOWNTIME_CREDITS_HOURS` in hours if it's only used in 1 place with `nodeMinLifespan`

Reply:
fixed